src/generalizedlinearmixedmodel.jl

```diff
@@ -346,6 +346,12 @@ function GeneralizedLinearMixedModel(
     end
     (isa(d, Normal) && isa(l, IdentityLink)) &&
     throw(ArgumentError("use LinearMixedModel for Normal distribution with IdentityLink"))
+
+    if !any(isa(d, dist) for dist in (Bernoulli, Binomial, Poisson))
+        @warn """Results for families with a dispersion parameter are known to be inaccurate.
```

Comment:
I might change that to "Results for families with a dispersion parameter are not reliable.  It is best to avoid trying to fit such models in MixedModels until the authors get a better understanding of those cases."